### PR TITLE
[15.0][MIG] sql_export

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Standard Migration to v15.
Moved from server-tools.

@JoanMForgeFlow

### CODE REVIEW

File: sql_export/readme/ROADMAP.rst [hunk 1]
```diff
@@ -15,4 +15,4 @@ See sql_request_abstract module to fix this issue.
   since variables features has been introduced. This can be fixed by
   overloading _prepare_request_check_execution() function.
 
-* Move modules sql_request_abstract and sql_export to oca/reporting-engine for version 15
+* * Moved from oca/server-tools version 14
```
Comment (legalsylvain) [inline sql_export/readme/ROADMAP.rst]: better to remove the line. the roadmap part is the remaing "things to do" IMHO.

**Review by legalsylvain — APPROVED** [review]

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM, Lets get this merged!

**Comment by sebalix** [comment]
Build is failing, can you check?

**Comment by yankinmax** [comment]
@mariadforgeflow can you pls check and include those changes from my PR?
https://github.com/OCA/server-tools/pull/2265/commits/2ce35948a0e9c8191fb9c15eb742b5e74e751cf2
https://github.com/OCA/server-tools/pull/2265/commits/0198d0c81ec7601b6bc7e7b3bb8c35f015fa638d
https://github.com/OCA/server-tools/pull/2265/commits/327697fab3b5f00e46d25590475ba2fd6fb1c112
**UPDATED**
or you can merge this PR and include changes to fix the build
https://github.com/ForgeFlow/reporting-engine/pull/3

**Comment by mariadforgeflow** [comment]
@yankinmax I think that the best way to proceed is to first merge the migration PR and then open a new one with improvements. I will now include the changes to fix the build:)

**Comment by yankinmax** [comment]
@mariadforgeflow as you wish, I'm here only to help to get it merged ASAP.
But, as you can see, you can cherry-pick my migration commit and it will be faster.
Thanks for quick response.

**Review by leemannd — APPROVED** [review]

**Comment by mariadforgeflow** [comment]
@yankinmax done:)

**Review by yankinmax — APPROVED** [review]

**Review by JoanMForgeFlow — APPROVED** [review]
green! LG

**Pull request merged**